js/server/modules/@arangodb/foxx/router/swagger-context.js

```diff
@@ -58,6 +63,19 @@ const repeat = (times, value) => {
   return arr;
 };
 
+function isNullSchema(schema) {
```

Comment:
This function is necessary because if users provide schemas there's no easy way to determine if the schema is just meant to say "don't provide a value for this" (e.g. request body must be omitted/empty).